packages/core/src/lib/plugins/AppsFlyerPlugin.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2020 Google Inc. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import {Plugin} from './Plugin';
+
+export const appsFlyerPlugin: Plugin = {
```

Comment:
I see how the `Plugin` class works well for AppsFlyer - do you know whether it's likely to work more generally as well? (If we wanted to add a new plugin, would it fit nicely into the existing class?)

Reply:
Yeah, we'll need to expand this as we implement more features.

Reply:
We'll need to add templates for the DelegationService too, and expand the Plugin (now renamed to Features) interface as we add new features.